src/components/FieldWrapper/FieldWrapper.tsx

```diff
@@ -36,11 +40,13 @@ export const FieldWrapper = React.forwardRef<HTMLDivElement, Props>(
             showArrow = false,
             isArrowUp = false,
             onArrowClick,
-            clearTooltipLabel = '',
+            clearTooltipLabel = CLOSE_ICON_LABEL,
             showClearButton = false,
             onClear = null,
             isFocused = false,
             disabled = false,
+            openArrowLabel = DOWN_ARROW_LABEL,
+            closeArrowLabel = UP_ARROW_LABEL,
```

Comment:
Can we see these constants in the Storybook as a default value?

Reply:
yes